samples/openapi3/server/petstore/spring-boot-oneof/src/main/java/org/openapitools/model/BarRef.java

```diff
@@ -26,6 +26,41 @@
 @Generated(value = "org.openapitools.codegen.languages.SpringCodegen")
 public class BarRef extends EntityRef implements BarRefOrValue {
 
+  public BarRef name(String name) {
+    super.setName(name);
+    return this;
+  }
+
+  public BarRef atReferredType(String atReferredType) {
+    super.setAtReferredType(atReferredType);
+    return this;
+  }
+
+  public BarRef href(String href) {
+    super.setHref(href);
+    return this;
+  }
+
+  public BarRef id(String id) {
+    super.setId(id);
+    return this;
+  }
+
+  public BarRef atSchemaLocation(String atSchemaLocation) {
+    super.setAtSchemaLocation(atSchemaLocation);
+    return this;
+  }
+
+  public BarRef atBaseType(String atBaseType) {
+    super.setAtBaseType(atBaseType);
```

Comment:
In order to not break the fluent setter chain, the inherited properties are exposed as a method in the child class.